modules/circe/README.md

```diff
@@ -20,10 +20,9 @@ import com.typesafe.config.ConfigFactory
 import pureconfig._
 import pureconfig.generic.auto._
 import pureconfig.module.circe._
-import pureconfig.error.CannotConvert
```

Comment:
You also need to fix this in the [tut source](https://github.com/pureconfig/pureconfig/blob/c9bc53861f5581b9b1d917f7d360d7b7ecb04fa5/modules/circe/src/main/tut/README.md).

Reply:
Rebased with this fix